src/HeckeMiscInteger.jl

```diff
@@ -0,0 +1,438 @@
+################################################################################
+#
+#  Integer functions
+#
+################################################################################
+
+function rem(a::ZZRingElem, b::UInt)
+    return ccall((:fmpz_fdiv_ui, libflint), UInt, (Ref{ZZRingElem}, UInt), a, b)
+end
+
+function isless(a::BigFloat, b::ZZRingElem)
+    if _fmpz_is_small(b)
+        c = ccall((:mpfr_cmp_si, :libmpfr), Int32, (Ref{BigFloat}, Int), a, b.d)
+    else
+        c = ccall((:mpfr_cmp_z, :libmpfr), Int32, (Ref{BigFloat}, UInt), a, unsigned(b.d) << 2)
+    end
+    return c < 0
+end
+
+function mulmod(a::UInt, b::UInt, n::UInt, ni::UInt)
```

Comment:
There are probably better places for this and other functions. What's the plan here? Move the first into this file and merge it; and then when we are done, we can take our time to move things to a nicer place?

Reply:
`src/flint/fmpz.jl`?